src/apply_args.rs

```diff
@@ -99,6 +99,9 @@ pub struct ApplyArgs {
     #[arg(from_global)]
     pub registry: Url,
 
+    #[arg(from_global)]
+    pub credentials: Vec<(String, String, String)>,
```

Comment:
You'll want to actually pass these in through the `configured_maintainer` function below.

Reply:
I'll just nod and do for now. To be honest: I haven't full understood yet how the whole flow with the arguments is. And the little I did understand is basically gone, now that I haven't looked at the code in 2 weeks.